packages/api-plugin-fulfillment-method-shipping-flat-rate/README.md

```diff
@@ -0,0 +1,70 @@
+# api-plugin-fulfillment-method-shipping-flat-rate
+
+[![npm (scoped)](https://img.shields.io/npm/v/@reactioncommerce/api-plugin-fulfillment-method-shipping-flat-rate.svg)](https://www.npmjs.com/package/@reactioncommerce/api-plugin-fulfillment-method-shipping-flat-rate)
+[![CircleCI](https://circleci.com/gh/reactioncommerce/api-plugin-fulfillment-method-shipping-flat-rate.svg?style=svg)](https://circleci.com/gh/reactioncommerce/api-plugin-fulfillment-method-shipping-flat-rate)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+
+## Summary
+
+This plugin provides the functionalities of a sample fulfillment method flat-rate under the fulfillment type of shipping.
+
+## Included in this example plugin
+
+### `src/`
```

Comment:
Again this is the example plugin templates, please replace with relevant data for this plugin

Reply:
Done